File: RepoTransBench/target_projects/Python/Go/stephenmcd_django-forms-builder/tests/original/utils_test.go
package original

import (
	"strings"
	"testing"

	"stephenmcd_django_forms_builder/formsbuilder"
)

func TestIsFileExtensions(t *testing.T) {
	if !formsbuilder.IsFile("photo.PNG") {
		t.Errorf("photo.PNG should be recognized as file")
	}
	if !formsbuilder.IsFile("document.PDF") {
		t.Errorf("document.PDF should be recognized as file")
	}
	if formsbuilder.IsFile("example.txt") {
		t.Errorf("example.txt should not be a file")
	}
	if formsbuilder.IsFile("no_dot") {
		t.Errorf("no_dot should not be a file")
	}
}

func TestSlugifyBasic(t *testing.T) {
	s := " Hello__World__ "
	sl := formsbuilder.Slugify(s)
	if sl != "hello-world" {
		t.Errorf("expected slug 'hello-world', got '%s'", sl)
	}
}

func TestIsEmailCases(t *testing.T) {
	if !formsbuilder.IsEmail("foo@bar.com") {
		t.Error("foo@bar.com should be recognized as email")
	}
	if formsbuilder.IsEmail("notanemail") {
		t.Error("notanemail should not be recognized as email")
	}
	if formsbuilder.IsEmail("@nodomain") {
		t.Error("@nodomain should not be recognized as email")
	}
}

func TestContentAsTxtHtml(t *testing.T) {
	text := formsbuilder.ContentAsText(123)
	html := formsbuilder.ContentAsHTML(struct{}{})
	if !strings.Contains(text, "text") {
		t.Error("ContentAsText should contain 'text'")
	}
	if !strings.Contains(html, "html") {
		t.Error("ContentAsHTML should contain 'html'")
	}
}

func TestGetAdminUrlFormat(t *testing.T) {
	url := formsbuilder.GetAdminURL(nil)
	if !(strings.Contains(url, "myapp/dummy/1")) {
		t.Errorf("expected url to contain myapp/dummy/1, got %s", url)
	}
}

File: RepoTransBench/target_projects/Python/Java/stephenmcd_django-forms-builder/src/test/java/com/stephenmcd/formsbuilder/original/UtilsTest.java
package com.stephenmcd.formsbuilder.original;

import com.stephenmcd.formsbuilder.utils.Utils;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

import java.util.*;

public class UtilsTest {

    @Test
    public void testIsFileExtensions() {
        assertTrue(Utils.isFile("photo.PNG"));
        assertTrue(Utils.isFile("document.PDF"));
        assertFalse(Utils.isFile("example.txt"));
        assertFalse(Utils.isFile("no_dot"));
    }

    @Test
    public void testSlugifyBasic() {
        String s = " Hello__World__ ";
        String sl = Utils.slugify(s);
        assertEquals("hello-world", sl);
    }

    @Test
    public void testIsEmailCases() {
        assertTrue(Utils.isEmail("foo@bar.com"));
        assertFalse(Utils.isEmail("notanemail"));
        assertFalse(Utils.isEmail("@nodomain"));
    }

    @Test
    public void testContentAsTxtHtml() {
        class Dummy {
            @Override
            public String toString() {
                return "dummyContent";
            }
        }
        String text = Utils.contentAsText(new Dummy());
        String html = Utils.contentAsHtml(new Dummy());
        assertTrue(text instanceof String);
        assertTrue(html instanceof String);
    }

    @Test
    public void testGetAdminUrlFormat() {
        class Dummy {
            // Simulate needed attributes
        }
        String url = Utils.getAdminUrl(new Dummy());
        assertTrue(url.contains("myapp/dummy/1/") || url.contains("myapp/dummy/1"));
    }
}